### PR TITLE
Update PowerShell commands

### PR DESCRIPTION
**Pull request by mslucyyloo22**

### CODE REVIEW

File: preview/MsixCore/README.md [hunk 1]
```diff
@@ -34,11 +34,15 @@ Sample packages can be found at msix-packaging/preview/MsixCore/tests; these can
...
-Installation - Using command prompt or PowerShell, navigate to the directory that contains the executables and run the following command to install notepadplus.msix.
+Installation - Using command prompt or PowerShell, navigate to the directory that contains the executables and run the following command to install notepadplus.msix. The -quietUX parameter can also be added at the end ofthe command to switch on the capability.
 
 ```
 msixmgr.exe -AddPackage C:\SomeDirectory\notepadplus.msix
 ```
+Querying for Installed MSIX Packages - 
+```
+msixmgr.exe -FindAllPackages
+```
```
Comment (wcheng-msft) [inline preview/MsixCore/README.md]: there's also a -FindPackage command

File: preview/MsixCore/README.md [hunk 2]
```diff
@@ -34,11 +34,15 @@ Sample packages can be found at msix-packaging/preview/MsixCore/tests; these can
 certutil -addstore root APPX_TEST_ROOT.cer
 ```
 
-Installation - Using command prompt or PowerShell, navigate to the directory that contains the executables and run the following command to install notepadplus.msix.
+Installation - Using command prompt or PowerShell, navigate to the directory that contains the executables and run the following command to install notepadplus.msix. The -quietUX parameter can also be added at the end ofthe command to switch on the capability.
```
Comment (wcheng-msft) [inline preview/MsixCore/README.md]: >ofthe [](start = 218, length = 5)

space

Comment (wcheng-msft) [inline preview/MsixCore/README.md]: >capability [](start = 249, length = 10)

mention what it does.

File: preview/MsixCore/README.md (hunk 1 above)
Comment (mslucyyloo22) [reply preview/MsixCore/README.md]: added rest of commands

**Pull request merged**